src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule21filename/InputOuterTypeFilename1.java

```diff
@@ -1,17 +1,17 @@
 package com.google.checkstyle.test.chapter2filebasic.rule21filename;
```

Comment:
It is required to remove the related suppression from the file  [checkstyle-input-suppressions.xml](https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle-input-suppressions.xml). 
Search for the filename in the mentioned file and delete that respective suppression. They are all organized directory wise, so it should be easy to browse.
Repeat the same for all 14 files that have been modified.